.circleci/config.yml

```diff
@@ -1,31 +1,53 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@0.1.0
+  plugin-ci: mattermost/plugin-ci@volatile
 
 workflows:
   version: 2
-  deploy:
+  nightly:
```

Comment:
I don't understand why we have a nightly lint/test/build scheduled here? Seems wasteful?

Reply:
Because our builds are not reproducible, there have been issues with broken `master` from time to time. A nightly helps us catch those issues.